sync.go

```diff
@@ -75,7 +75,10 @@ func (s *Syncer) bgRemove(ctx context.Context, removeCh chan *Target, addCh chan
 			removeUnixTime := now.Add(s.Config.RemoveDelay).Unix()
 			if headItem, headAt := q.Head(); headItem == nil || removeUnixTime < headAt {
 				if !t.Stop() {
-					<-t.C
+					select {
+					case <-t.C:
+					default:
```

Comment:
Can we just remove this block if we're going to fall through, whether there's something in the channel or not?

Reply:
This is to clear the channel as describedin https://golang.org/pkg/time/#Timer.Reset